test/components/form/submissions.spec.js

```diff
@@ -337,6 +338,21 @@ describe('FormSubmissions', () => {
               tableSelector: `#form-submissions-table${i}`
             }));
       }
+
+      it('disables the download button', () =>
+        loadSubmissions(1)
+          .afterResponses(component =>
+            component.first('#form-submissions-download-button')
+              .hasClass('disabled').should.be.false())
+          .request(component => trigger.click(component, '.btn-refresh'))
+          .beforeEachResponse(component =>
+            component.first('#form-submissions-download-button')
+              .hasClass('disabled').should.be.true())
```

Comment:
given you just added custom assertions for visible-or-not perhaps disabled deserves one too?

Reply:
Ah good idea. The assertion would have to be smart, looking at the `disabled` attribute for buttons and the `disabled` class for links, but I think it'd make these tests more readable. I think I'll add that the next time I test whether an element is disabled.